src/AggregateSpecsPlugin.ts

```diff
@@ -48,10 +52,13 @@ const AggregateSpecsPlugin: Plugin = (builder) => {
         id: "sum",
         humanLabel: "sum",
         HumanLabel: "Sum",
-        isSuitableType: isNumberLike,
+        isSuitableType: (pgType: PgType): boolean =>
+          isNumberLike(pgType) || pgType.id === INTERVAL_OID,
         // I've wrapped it in `coalesce` so that it cannot be null
-        sqlAggregateWrap: (sqlFrag) =>
-          sql.fragment`coalesce(sum(${sqlFrag}), 0)`,
+        sqlAggregateWrap: (sqlFrag, pgType) =>
```

Comment:
Added `pgType` as an argument to `sqlAggregateWrap` so `coalesce` can provide the correct default for intervals. Seems like it's generally useful to have this here. It might even make sense to provide the whole attribute object.

Reply:
We can't pass the attribute because the aggregate doesn't always apply to an attribute; but passing the type definitely makes sense. I used to have this but removed it because nothing used it :grin: I'm happy with it being re-added.